### PR TITLE
Aligned the list and Resolved the warnings raised in the console

### PR DESCRIPTION
**Pull request by vikkastiwari**
FROM -

TO -
Capture

Resolved the warnings raised in the console
`src\components\AddItem.js
  Line 7:53:   Expected '!==' and instead saw '!='  eqeqeq
  Line 13:32:  Expected '!==' and instead saw '!='  eqeqeq`

### CODE REVIEW

**Comment by code-knackerin** [comment]
Hey, can you please just for responsive ness. thx

**Comment by vikkastiwari** [comment]
> Hey, can you please just for responsiveness. thx

Yes, I checked for responsiveness it works in mobile view as well.

File: src/App.css [hunk 1]
```diff
@@ -30,4 +30,5 @@ input[type="text"] {
   grid-template-columns: 1fr 1fr 1fr;
   grid-template-rows: 1fr 1fr 1fr;
   gap: 15px 15px;
+  margin-right: 31px;
```
Comment (m1ga) [inline src/App.css:33]: This is not the right way to do it. Why adding "random" margin on the right side?
Check the page/structure: it is a `<ul>` so it has a default left padding. The better way to center the elements is:
```
.boxForItems{ padding: 0}
.foodListItems {margin:auto}
```
(just adding that to the existing styles)

Comment (vikkastiwari) [reply src/App.css:33]: Okay, I will work on this.
Thank you for your guidance.

File: src/App.css [hunk 2]
```diff
@@ -30,4 +30,7 @@ input[type="text"] {
   grid-template-columns: 1fr 1fr 1fr;
   grid-template-rows: 1fr 1fr 1fr;
   gap: 15px 15px;
+  padding: 0;
 }
+
+.foodListItems {margin:auto}
```
Comment (m1ga) [inline src/App.css:36]: no, add it here: https://github.com/code-knackerin/ShoppingList/blob/main/src/components/List.css
`foodListItems` is in a different component.

Comment (vikkastiwari) [reply src/App.css:36]: okay

Comment (vikkastiwari) [reply src/App.css]: > no, add it here: https://github.com/code-knackerin/ShoppingList/blob/main/src/components/List.css `foodListItems` is in a different component.

Done

**Comment by code-knackerin** [comment]
Hey guys Thanks for your work! And m1ga thanks for your awesome help. Its so kind from you! :-)
Happy Hacking and a nice day. 
H.

**Pull request merged**